lms/envs/bok_choy.py

```diff
@@ -232,16 +232,7 @@
         'course_author': 'http://edx.readthedocs.io/projects/edx-partner-course-staff',
     }
 
-# TODO: TNL-6546: Remove this waffle and flag code.
-from django.db.utils import ProgrammingError
-from waffle.models import Flag
-try:
-    flag, created = Flag.objects.get_or_create(name='unified_course_view')
-    WAFFLE_OVERRIDE = True
-except ProgrammingError:
-    # during initial reset_db, the table for the flag doesn't yet exist.
-    pass
-
+WAFFLE_OVERRIDE = True
```

Comment:
@robrap, you wrote most of this block, does this removal look okay to you? Would you prefer I leave the TODO in-place for WAFFLE_OVERRIDE?

Context: I have to update the bok choy db cache for this PR, which involves adding a `unified_course_view` waffle flag. Since this block is run *before* the db cache is loaded, I end up trying to insert it twice and get an IntegrityError ([example](https://build.testeng.edx.org/job/edx-platform-test-subset/816707/console)).